docs/tutorials/deploy-vm/iso.md

```diff
@@ -0,0 +1,165 @@
+      MDMxCzAJBgNVBAYTAlVTMRYwFAYDVQQKEw1MZXQncyBFbmNyeXB0MQwwCgYDVQQD
+      EwNSMTEwHhcNMjQxMjA2MTE1MzAzWhcNMjUwMzA2MTE1MzAyWjAhMR8wHQYDVQQD
+      ExZtaXJyb3JzLnJvY2t5bGludXgub3JnMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8A
+      MIIBCgKCAQEA6EZ30U0n8CBlWl8jC3Zwasky+3SQDXmehkPLo0K0v7kHPE4g+NUa
+      kvyanUHYuGtudvleMTbZSvZp7bv8iFRll41r0Z01j49vVEIsL5wixb4NyJX2DDne
+      a0G3ta+JzrNQhCqcKuXKFo4EgAYTvaKSa1KEZ+REUb8XhtafUyImSehF0xYWE1Tc
+      obyg57oCO+SXfP9ypHjCC9/rZjbrU/UA+x+Sofydt1HkQ4g/dXX3prNnD75k8ue6
+      3MkyP2QcfFc3IfzsoGFUXxtm10F6BGjwGm289qxSuIqcX7ylmMnCVGs4q+UrPWPN
+      bXWDU7ItnOlGjML000+l8p500iFSXcQO3QIDAQABo4ICTTCCAkkwDgYDVR0PAQH/
+      BAQDAgWgMB0GA1UdJQQWMBQGCCsGAQUFBwMBBggrBgEFBQcDAjAMBgNVHRMBAf8E
+      AjAAMB0GA1UdDgQWBBSGel3Hh031yt91oQYM89+kOLE7DTAfBgNVHSMEGDAWgBTF
+      z0ak6vTDwHpslcQtsF6SLybjuTBXBggrBgEFBQcBAQRLMEkwIgYIKwYBBQUHMAGG
+      Fmh0dHA6Ly9yMTEuby5sZW5jci5vcmcwIwYIKwYBBQUHMAKGF2h0dHA6Ly9yMTEu
+      aS5sZW5jci5vcmcvMFQGA1UdEQRNMEuCF2Rvd25sb2FkLnJvY2t5bGludXgub3Jn
+      ghhkb3dubG9hZHMucm9ja3lsaW51eC5vcmeCFm1pcnJvcnMucm9ja3lsaW51eC5v
+      cmcwEwYDVR0gBAwwCjAIBgZngQwBAgEwggEEBgorBgEEAdZ5AgQCBIH1BIHyAPAA
+      dQDgkrP8DB3I52g2H95huZZNClJ4GYpy1nLEsE2lbW9UBAAAAZOcBojlAAAEAwBG
+      MEQCIF2OTCyynecFhDAg7Im7bxs0s1W5ieZxCp0wM/0CJ7HgAiB4gPxrdXVprVmR
+      49faW7Lqhw8mfzOCQV7vyKN9CIVJBgB3AM8RVu7VLnyv84db2Wkum+kacWdKsBfs
+      rAHSW3fOzDsIAAABk5wGiLMAAAQDAEgwRgIhAORV5PFPjCPkuX3zG3S1cbKwGU8n
+      MkKDgjJyMmUO9cSGAiEAzmvrDHTwJIV4AeGWFzMO3adyo0pWOCnEoIcVqLwWk8Yw
+      DQYJKoZIhvcNAQELBQADggEBAKm1NbsCgzov+GYAZHkkOHcw6ypbTsg1126HIB6k
+      n4RKFDTpvj62YCZTkTSAKY7gHSMt18jNN7LWv344dxLSB+goA5QSMUNHrsURoRW3
+      Bh7n2j6rr6LSF2HhBvRBPUk8HI8OgvuBXP0looxVgfBwybCEwjCa6iTaqcwPcECz
+      8P5r+lRn2nj3aDFrQ/zYZjA28A2bIgjcedg/5q1FNwPZWzsQ/tK6wkDeENzLgs5x
+      y9AY7M6fnRUbYGYauLA50clZAmxG63oApiaOvNFn1oK3TSAHkk1mZ16ibupI0e2F
+      Ypmf48/C4TvbDkL8qGM0o8hDqzax8eEY4w6sWbV9U/HnzcM=
+      -----END CERTIFICATE-----
+    checksum: # (3)
+      value: ba60c3653640b5747610ddfb4d09520529bef2d1d83c1feb86b0c84dff31e04e
+      algorithm: SHA256
+  target:
+    item:
+      name: rocky-dvd-iso
+      type: ISO
+    library:
+      apiVersion: imageregistry.vmware.com/v1alpha1
+      kind: ContentLibrary
+      name: cl-bb87952414e9859e8 # (4)
+```
+
+1.  :wave: The URL specifies an ISO file to be imported as a `VirtualMachineImage` resource.
+2.  :wave: A PEM encoded SSL Certificate for the endpoint specified by the above URL.
+3.  :wave: An optional field to verify the checksum algorithm and value for the file at the URL.
+4.  :wave: The name of the Kubernetes content library object associated with the namespace for importing the ISO image.
+
+## Deploy VM from ISO
+
+Once the ISO image is available, you can deploy a VM with a CD-ROM configured to mount the ISO image and a PVC (PersistentVolumeClaim) attached as the VM's boot disk. The following example demonstrates how to create both the PVC and VM objects for an ISO-based deployment:
```

Comment:
Try to avoid second-person, colloquial language. Change it `Once the ISO image is available, it can be deployed...`

Reply:
Rephrased as suggested, thanks!